### PR TITLE
Animations + sticky scroll

### LINKED ISSUE #33: Animate navigation UI
Add animations for some of our site components as you see fit. 
Perhaps make the About page user cards flip? A cool transition from page to page? 

Open to discussion.

### LINKED ISSUE #35: Snap-to-Top-Button
Add a sticky button that persists as a user scrolls down (hence 'sticky'), and upon click, scrolls the user back up.

### PR DESCRIPTION
**Pull request by julianp15**
Closes #33 
Closes #35

### CODE REVIEW

File: client/src/Layout.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { Link, useLocation } from "react-router-dom";
-import { Footer } from "./components/Footer";
```
Comment (julianp15) [inline client/src/Layout.jsx:2]: THis should not be removed sorry

Comment (jpatterson933) [reply client/src/Layout.jsx:2]: @julianp15 You can fix it and repush the branch and it will update here

File: client/src/Layout.jsx [hunk 2]
```diff
@@ -47,9 +57,25 @@ const Layout = ({ children }) => {
       </nav>
 
       {children}
-      {/* This is the footer component located on /component/Footer/index.js */}
-      <Footer />
```
Comment (jpatterson933) [inline client/src/Layout.jsx:51]: This is the same footer component that should still be there.

File: client/src/Layout.jsx [hunk 3]
```diff
@@ -47,9 +57,25 @@ const Layout = ({ children }) => {
...
+
+   
+
+      <footer id="#footer-bar">
```
Comment (jpatterson933) [inline client/src/Layout.jsx]: `<Footer />` would go here

**Review by jpatterson933 — COMMENTED** [review]
Everything looks good except for the Footer component. that just needs to be fixed and added back into its original place.

File: client/src/Layout.jsx [hunk 4]
```diff
@@ -49,7 +60,23 @@ const Layout = ({ children }) => {
       {children}
       {/* This is the footer component located on /component/Footer/index.js */}
       <Footer />
+
+      <footer id="#footer-bar">
```
Comment (StephenODea54) [inline client/src/Layout.jsx]: This element needs removed, otherwise all good!

**Review by StephenODea54 — CHANGES_REQUESTED** [review]

**Review by StephenODea54 — APPROVED** [review]
Awesome, looks good!

**Review by jpatterson933 — APPROVED** [review]
Good fix!

**Comment by nathanjslim** [comment]
@julianp15 Clicking the title screen does not return us to home page. Not sure if you attempted to address this in Layout.jsx or not; could you revisit and add this?

If you're too busy I can fix this as well. Let me know

**Pull request merged**